lightning-network-book.tex

```diff
@@ -40,7 +40,7 @@ \chapter{Introduction}
 We continue by explaining the theory and construction of the Lightning Network.
 We will understand how to construct payment channels via Revocable Sequence Maturity Contracts and how to solve the routing issues via Hashed Time Locked Contracts.
 These two forms of smart contracts allow for the Blockchain to be transformed from a transaction layer for payments to an enforcing layer of those contracts.
-Similar to the real world most contracts have never to be enforced via a court ruling but are negotiated bilaterally between consenting parties.
+Similar to the real world, most contracts will never to be enforced via a court ruling but are negotiated bilaterally between consenting parties.
```

Comment:
is that actually true? When including the word `will` I would write: 
... will never  be enforced...
omitting the word to